subprojects/docs/src/docs/userguide/build_cache.adoc

```diff
@@ -30,7 +30,7 @@ When using a shared build cache for task output caching this even works across d
 
 Apart from task output caching, we expect other features to use the build cache in the future.
 
-NOTE: A complete guide is available about https://guides.gradle.org/using-build-cache/[using the build cache]. It covers the different scenarios caching can improve, and detailed discussions of the different caveats you need to be aware of when enabling caching for a build.
+TIP: For a hands-on approach to learning how to use the build cache, try the https://guides.gradle.org/using-build-cache/[Using the Build Cache] guide. It covers the different scenarios that caching can improve and has detailed discussions of the different caveats you need to be aware of when enabling caching for a build.
```

Comment:
This promotes the tutorial a bit more firmly to make up for its removal from the left-hand nav.